wordpress-mobile/gutenberg-mobile
# Hotfix v1.15.2

PR to merge hotfix v1.15.2 into master

- `Gutenberg` PR: https://github.com/WordPress/gutenberg/pull/18128
- `WP-Android` PR: https://github.com/wordpress-mobile/WordPress-Android/pull/10706
- `WP-iOS` PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/12783

To test:
- [iOS] Upload media (Image, Video, Media&Text Blocks).
- [iOS] Copy images from Photos app into gutenberg editor.
- [Android] Scroll position (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1478)
- Block inserter layout should be correct after device rotation. (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1495)
- Added left right borders to inner blocks (https://github.com/wordpress-mobile/gutenberg-mobile/pull/1496)
- Removed alignment options from Media & Text until they are fixed (#1500)


**UPDATE:** Known issues to be solved on a next hotfix/release:

~- https://github.com/wordpress-mobile/gutenberg-mobile/issues/1509~
~- https://github.com/wordpress-mobile/gutenberg-mobile/issues/1510~


Update release notes:
